exercises/toRoman/toRoman.js

```diff
@@ -15,14 +15,47 @@
  * @returns {string} The input integer's Roman numeral representation
  */
 function toRoman(num) {
-  // This is your job. :)
+  let ones = ["I","IV","V","VI","VII","VIII","IX","X"];
```

Comment:
The `README` suggest you get the program working without handling numbers like `IV` and `IX`. I'd follow it!